dto/src/main/java/org/jboss/pnc/dto/response/Banner.java

```diff
@@ -21,10 +21,18 @@
 import lombok.Setter;
 import lombok.ToString;
 
+/**
+ * Announcement banner. This banner is used to notify users about important information.
+ * 
+ * @author jbrazdil
+ */
 @ToString
 @Getter
 @Setter
 public class Banner {
 
+    /**
+     * Banner text.
```

Comment:
Is this just plain text or can this be html?

Reply:
No idea. @thescouser89 ?

Reply:
plain text